recipes/hsp2/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "hsp2" %}
+{% set version = "0.11.0a1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/hsp2-{{ version }}.tar.gz
+  sha256: 53b53f623f0e6c96bb9dbcb23d85377fd39486c9889eac7ecd6d3227cd063a74
+
+build:
+  entry_points:
+    - hsp2 = hsp2.hsp2tools.HSP2_CLI:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=45
+    - wheel
+    - numpy <2.0
```

Comment:
Normally, you don't need `numpy` in host if the package is noarch
```suggestion
```

Reply:
BTW, while it is declared upstream, see https://github.com/respec/HSPsquared/blob/6f40cc30cd9ed771d26501b5249c0353e3cdc051/pyproject.toml#L6, I believe that is wrong b/c I could not see any compiled extension that would require numpy. You should probably fix that upstream too so the automated tools can work properly.

Reply:
Thank you.  I will make that change.